content/php/concepts/date-functions/terms/date-add/date-add.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'date_add()'
+Description: 'Adds a given number of days, months, years, hours, minutes or seconds to an existing DateTime object.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Time'
+  - 'Objects'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`date_add()`** function takes an existing DateTime object and adds a specified number of days, months, years, hours, minutes or seconds.
+
+## Syntax
+
+```pseudo
+date_add(date_object, time_frame_to_add)
+```
+
+The `date_object` parameter is an existing DateTime object.
+
+The `time_frame_to_add` parameter is a DateInterval object.
+
+## Codebyte Example
```

Comment:
I've only included a codebyte example as there are a lot of different aspects that could be altered to change the output. Is it ok to have just a codebyte example? I wasn't sure if this should be changed to a standard example, instead of a codebyte, or if a standard example should be added as well?

Reply:
The default is to include a standard example and a codebyte is optional. A codebyte has value because it's interactive, but it should share something not evident in the previous example, we don't want to have examples that  are redundant.

Reply:
Ok, thanks for explaining. I have changed the codebyte to a standard example.